sbin/common-functions.sh

```diff
@@ -100,7 +100,7 @@ checkingAndDownloadCaCerts()
   git clone https://github.com/AdoptOpenJDK/openjdk-build.git cacerts_area
   echo "cacerts should be here..."
   # shellcheck disable=SC2046
-  if [ $(file "${WORKING_DIR}/cacerts_area/security/cacerts") -ne 0 ]; then
+  if ! (file "${WORKING_DIR}/cacerts_area/security/cacerts"); then
```

Comment:
The brackets aren't _strictly_ necessary here, these could all just be 

```bash
   if ! file "${WORKING_DIR}/cacerts_area/security/cacerts"; then 
```

For the longer ones like:

```bash
     if ! (bash ./configure --prefix="${WORKING_DIR}"/"${OPENJDK_REPO_NAME}"/installedfreetype "${FREETYPE_FONT_BUILD_TYPE_PARAM}" && $MAKE all && $MAKE install); then 
```

I think the brackets probably make things clearer (especially for people not familiar with bash), but if it's a simple command it can just be `if command; then`

Reply:
Yes agree and kinda marks demarkation between the condition and if ... then syntax